proto/wechaty/puppet/location.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+package wechaty.puppet;
+
+option go_package       = "github.com/wechaty/go-grpc/wechaty/puppet";
+option java_package     = "io.github.wechaty.grpc.puppet";
+option csharp_namespace = "github.wechaty.grpc.puppet";
+
+message LocationPayload {
+  uint32 accuracy = 1;
```

Comment:
Never use `uint32`, use `int32` instead.

> unsigned integer types such as uint32 and fixed32 should not be used because some important programming languages and systems don't support them well
&mdash; https://cloud.google.com/apis/design/design_patterns